src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -10,6 +10,7 @@ public static class RpcWorkerConstants
         public const string FunctionWorkerRuntimeSettingName = "FUNCTIONS_WORKER_RUNTIME";
         public const string FunctionWorkerRuntimeVersionSettingName = "FUNCTIONS_WORKER_RUNTIME_VERSION";
         public const string FunctionsWorkerProcessCountSettingName = "FUNCTIONS_WORKER_PROCESS_COUNT";
+        public const string FunctionsWorkerSharedMemoryDataTransferSettingName = "FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER";
```

Comment:
I think this should be renamed FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED to make it clear how to set it

Reply:
Once we agree on how to keep this enabled/disabled by default, then we can finalize this. We can either follow what you said or the opposite `FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_DISABLED`.

Reply:
For now,  please change to `FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED`